### PR TITLE
[Agent Guide] Integration Management

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
- Add Agent Guide on Integration Management

### Motivation
- Trello request

### Preview link
https://docs-staging.datadoghq.com/ruth/integration-mgmt/agent/guide/

### CODE REVIEW

File: content/en/agent/guide/integration-management.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+title: Integration Management
+kind: guide
+---
+
+## Overview
+
+The Agent comes with a set of bundled official Datadog integrations to allow users to quickly start monitoring their applications. These integrations are also available as single Python packages that can be individually upgraded.
```
Comment (apigirl) [inline content/en/agent/guide/integration-management.md]:
```suggestion
The Agent comes with a set of bundled official Datadog integrations to allow users to start monitoring their applications quickly. These integrations are available as single Python packages, and you can upgrade them separately.
```

Comment (apigirl) [reply content/en/agent/guide/integration-management.md]: the first sentence is a split infinitive and the second sentence is passive voice that I think could a bit more clear when made active

File: content/en/agent/guide/integration-management.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+For Agent v6.8+, the `datadog-agent integration` command allows users to manage the official Datadog integrations that are available for the Agent. It has the following subcommands:
+
+ * [install](#install)
+ * [remove](#remove)
+ * [show](#show)
+ * [freeze](#freeze)
+
+The usage and documentation of these commands can be printed with `datadog-agent integration --help`.
```
Comment (apigirl) [inline content/en/agent/guide/integration-management.md]:
```suggestion
Print the usage and documentation of these commands with `datadog-agent integration --help`.  
```

Comment (apigirl) [reply content/en/agent/guide/integration-management.md]: also passive - making it not passive makes it a bit more concise

File: content/en/agent/guide/integration-management.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+## Integration commands
+
+### Summary workflow
+
+1. Check the version of the integration installed in your Agent with the `show` command.
+2. Review the changelog of the specific integration on the [integrations-core][1] repository to identify the version you want.
+3. Install the integration with the `install` command.
+4. Restart your Agent.
+
+**Note**: When using a configuration management tool, it is recommended to pin the integration to the desired version. When the Agent is upgraded, remove the integration pin. Upgrading the Agent without removing the integration pin can cause the configuration management tool to fail if the version of the integration is not compatible with the new version of the Agent.
```
Comment (apigirl) [inline content/en/agent/guide/integration-management.md]:
```suggestion
**Note**: When using a configuration management tool, it is recommended to pin the integration to the desired version. When you're ready to upgrade the Agent, remove the pin. Upgrading the Agent without removing the integration pin can cause the configuration management tool to fail if the version of the integration is not compatible with the new version of the Agent.
```

**Comment by ruthnaebeck** [comment]
@trishankatdatadog @dsahni - Let us know if this is ok to merge.

File: content/en/agent/guide/integration-management.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+ * [remove](#remove)
+ * [show](#show)
+ * [freeze](#freeze)
+
+Print the usage and documentation of these commands with `datadog-agent integration --help`.  
+For Linux, execute the command as the `dd-agent` user. For Windows, execute the command as an `administrator`.
+
+## Integration commands
+
+### Summary workflow
```
Comment (dsahni) [inline content/en/agent/guide/integration-management.md]:
```suggestion
### Workflow
```

File: content/en/agent/guide/integration-management.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+**Note**: When using a configuration management tool, it is recommended to pin the integration to the desired version. When you're ready to upgrade the Agent, remove the pin. Upgrading the Agent without removing the integration pin can cause the configuration management tool to fail if the version of the integration is not compatible with the new version of the Agent.
+
+### Install
+
+Use the `datadog-agent integration install` command to install a specific version of an official Datadog integration (available on the [integrations-core repository][1]), provided that it is compatible with the version of the Agent. The command does this verification and exits with a failure in case of incompatibilities.
+
+An integration is compatible and installable if:
+ 1. The version is newer than the one [shipped with the Agent][2].
+ 1. It is compatible with the version of the [datadog_checks_base][3].
```
Comment (dsahni) [inline content/en/agent/guide/integration-management.md]: `It is compatible with the version of the datadog_checks_base`

Which version? Current version available, or the version installed on the host? This sentence isn't very clear.

Comment (ruthnaebeck) [reply content/en/agent/guide/integration-management.md]: @dsahni - This was taken from the docs in the `datadog-agent` repo. We would need to check with the engineers.

Comment (zippolyte) [reply content/en/agent/guide/integration-management.md]: version available in the installed agent

**Comment by zippolyte** [comment]
LGTM, just answered @dsahni's question

**Pull request merged**